kolibri/plugins/setup_wizard/api.py

```diff
@@ -32,6 +32,141 @@ def has_permission(self, request, view):
         return subset_of_users_device
 
 
+class SetupWizardResource(ViewSet):
```

Comment:
This appears to be replacing the core provision device endpoint: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/device/api.py#L78

which has decent test coverage of different pathways: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/device/test/test_api.py#L50

I would much rather we add these conditionals to that endpoint and test them, rather than creating a brand new endpoint here.